BuildResidentialHPXML/measure.rb

```diff
@@ -3452,13 +3522,22 @@ def self.set_walls(hpxml, runner, model, args)
 
       next if exterior_adjacent_to == HPXML::LocationLivingSpace # already captured these surfaces
 
+      attic_wall_type = nil
+      if ([HPXML::LocationAtticUnvented, HPXML::LocationAtticVented].include? interior_adjacent_to) && (exterior_adjacent_to == HPXML::LocationOutside)
```

Comment:
Should probably include `HPXML::LocationAtticUnconditioned` (as you did above) just to be comprehensive.

Reply:
Addressed in a few places